packages/eas-cli/src/credentials/ios/appstore/authenticate.ts

```diff
@@ -18,22 +21,31 @@ import {
 
 const APPLE_IN_HOUSE_TEAM_TYPE = 'in-house';
 
+export enum AuthenticationMode {
+  API_KEY,
+  USER,
```

Comment:
```suggestion
  /** Uses cookies based authentication and the unofficial App Store web API, this provides more functionality than the official API but cannot be reliably used in CI because it requires 2FA. */
  USER,
```